mmv1/products/monitoring/terraform.yaml

```diff
@@ -147,6 +147,28 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       custom_import: templates/terraform/custom_import/self_link_as_name.erb
       encoder: templates/terraform/encoders/monitoring_service.go.erb
 
+  GenericService: !ruby/object:Overrides::Terraform::ResourceOverride
+    legacy_name: 'google_monitoring_service'
+    id_format: "{{name}}"
```

Comment:
ID Format and import format should use the projects/{{project}}/.... form instead of just {{name}}

Reply:
👍